package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "soap",
-  "version": "0.8.0",
+  "version": "0.8.1",
```

Comment:
Please revert your version change as we can't guarantee that our next release will be `0.8.1` or `0.9.0` or something the like. Please just change the `package.json` if it's necessary because of `dependencies`, `devDependencies` or `scripts`.